snuba/admin/views.py

```diff
@@ -708,3 +711,109 @@ def snql_to_sql() -> Response:
             400,
             {"Content-Type": "application/json"},
         )
+
+
+@application.route("/allocation_policies")
+@check_tool_perms(tools=[AdminTools.CAPACITY_MANAGEMENT])
+def allocation_policies() -> Response:
+    return Response(
+        json.dumps(get_allocation_policies()), 200, {"Content-Type": "application/json"}
+    )
+
+
+@application.route("/allocation_policy_configs/<path:storage>", methods=["GET"])
+@check_tool_perms(tools=[AdminTools.CAPACITY_MANAGEMENT])
+def get_allocation_policy_configs(storage: str) -> Response:
+    try:
+        policy = get_storage(StorageKey(storage)).get_allocation_policy()
+        configs = policy.get_configs()
+        return Response(json.dumps(configs), 200, {"Content-Type": "application/json"})
+    except Exception as exception:
+        return Response(
+            json.dumps({"error": {"message": str(exception)}}, indent=4),
+            400,
+            {"Content-Type": "application/json"},
+        )
+
+
+@application.route(
+    "/allocation_policy_parameterized_config_definitions/<path:storage>",
+    methods=["GET"],
+)
+@check_tool_perms(tools=[AdminTools.CAPACITY_MANAGEMENT])
+def get_allocation_policy_parameterized_config_definitions(storage: str) -> Response:
```

Comment:
why does this need a separate endpoint?

Reply:
This is kind of for answering the question "What optional configs do we have and how are they defined?".

This informs the "Add new" modal with what can be configured and what it expects for types of the parameters etc. I'll change the "parameterized" nomenclature to "optional".